docs/quickstart.md

```diff
@@ -0,0 +1,766 @@
+---
+title: QuickStart
+description: Powertools introduction
+---
+Quickstart introducing core Powertools functionalities.
+
+
+## Installation 
+With [SAM](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) installed, you can follow this Quickstart step by step or you can build a project with the final Quickstart code you can play with.
+If you follow it all along, create a new empty project.
+=== "shell"
+```bash
+sam init --runtime python3.9 --dependency-manager pip --app-template hello-world --name powertools-quickstart
+```
+If you want to play with the final code, then you can download it from [aws-samples](https://github.com/aws-samples/cookiecutter-aws-sam-python) github repository.
+=== "shell"
+```bash
+sam init --location https://github.com/aws-samples/cookiecutter-aws-sam-python
+```
+### Configuration
+If you deploy code from Quickstart make sure to [set up your AWS credentials](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-getting-started-set-up-credentials.html).
+
+### Code example
+In the `powertools-quickstart` folder, we modify the following files:
+
+* **app.py** - Application code.
+* **template.yaml** - AWS infrastructure configuration using SAM.
+* **requirements.txt** - List of extra Python packages needed.
```

Comment:
Was this perhaps a leftover? This is not being changed. I suspect you had it at some point, then used Lambda Layers, then removed, etc.

Paths you can take:

1. Include Lambda Powertools Layer (preferred) in the template
2. Include the change necessary within requirements.txt

In either option, they will have to create a virtual env & install Lambda Powertools as a development dependency in order to gain auto-complete / IntelliSense in their IDE.

Reply:
We modify requirement.txt file in *API Gateway router* section when we introduce powertools. My intention here was to give a summary what files we change as we progress with our journey. Will make it more explicit to avoid confusion

Reply:
Modified this piece of documentation to this:
```
### Project structure
As we move forward, we will modify the following files within the `powertools-quickstart` folder:

* **app.py** - Application code.
* **template.yaml** - AWS infrastructure configuration using SAM.
* **requirements.txt** - List of extra Python packages needed.
```
Let me know if this is ok

Reply:
Looks awesome! thank you!